docs/README.md

```diff
@@ -22,9 +22,10 @@ Previewing the docs requires the [App Engine Standard Environment for Python 2.7
 
 2. Install npm dependencies for the docs site.
 
-     cd lit-html/docs
-     npm install
-
+```
```

Comment:
Can you indent the whole block 4 spaces so it lines up under the step?

Reply:
Done.